src/foremast/awslambda/api_gateway_event/api_gateway_event.py

```diff
@@ -151,7 +151,7 @@ def add_permission(self):
             region=self.region,
             source_arn=global_api_source_arn)
 
-    @retries(max_attempts=5, wait=2, exceptions=(botocore.exceptions.ClientError))
+    @retries(max_attempts=10, wait=6, exceptions=(botocore.exceptions.ClientError))
```

Comment:
same comment as security_group.py

Reply:
I have replaced this with the backoff library per my initial recommendation in my pull request. I have changed the max attempts back to the original values per request